README.md

```diff
@@ -65,7 +65,11 @@ Installation using [bower](http://bower.io):
 
     $ bower install <package-name> , where <package-name> is one of the package names mentioned above, for example:
 
-    $ bower install cldr-dates-full
+    $ bower install cldr-dates-
```

Comment:
The package name cldr-dates-full got chopped. Probably want to fix

Reply:
```suggestion
    $ bower install cldr-dates-full
```

^ ?